packages/polaris-viz/src/components/ComparisonMetric/ComparisonMetric.tsx

```diff
@@ -15,41 +14,26 @@ export function ComparisonMetric({
   trend,
   accessibilityLabel,
 }: Omit<ComparisonMetricProps, 'dataIndex'>) {
-  const selectedTheme = useTheme();
-
+  let direction;
   switch (trend) {
-    case 'neutral':
-      return (
-        <span className={styles.NeutralIcon}>
-          <span className={styles.VisuallyHidden}>{accessibilityLabel}</span>
-          <span style={{color: selectedTheme.legend.trendIndicator.neutral}}>
-            -
-          </span>
-        </span>
-      );
     case 'positive':
-      return (
-        <span className={styles.PositiveIcon}>
-          <UpChevron
-            accessibilityLabel={accessibilityLabel}
-            fill={selectedTheme.legend.trendIndicator.positive}
-          />
-          <span style={{color: selectedTheme.legend.trendIndicator.positive}}>
-            {metric}
-          </span>
-        </span>
-      );
+      direction = 'upward';
+      break;
     case 'negative':
-      return (
-        <span className={styles.NegativeIcon}>
-          <DownChevron
-            accessibilityLabel={accessibilityLabel}
-            fill={selectedTheme.legend.trendIndicator.negative}
-          />
-          <span style={{color: selectedTheme.legend.trendIndicator.negative}}>
-            {metric}
-          </span>
-        </span>
-      );
+      direction = 'downward';
+      break;
+    case 'neutral':
```

Comment:
@envex just wanted to confirm we don't want to display any arrows for the "neutral" trend. We want to display `–`?

Reply:
What about displaying nothing. Here for example, maybe it makes sense to not render the `-` at all

![image](https://github.com/user-attachments/assets/2cd118fb-e811-4d98-b742-6204ab5585a4)

Reply:
I think we should keep it as is for now but worth thinking about in the future. Removing it and displaying nothing will create odd spacing/padding when other values have a negative/positive trend.